src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import '../../App.scss';
```

Comment:
you dont need to add css file here, its already decalred in App.jsx file

Reply:
agree, but it's no problem for scss

Reply:
Done